src/test/java/com/conveyal/datatools/manager/controllers/api/UserControllerTest.java

```diff
@@ -62,6 +65,73 @@ public static void tearDown() {
         UserController.setBaseUsersUrl(UserController.DEFAULT_BASE_USERS_URL);
     }
 
+    /**
+     * Make sure the user endpoint can return a list of users
+     */
+    @Test
+    public void canListFirstTenUsers() throws IOException {
+        // create wiremock stub for get users endpoint
+        stubFor(
+            get(urlPathEqualTo("/api/v2/users"))
+                .withQueryParam("page", equalTo("1"))
+                .willReturn(
+                    aResponse()
+                        .withBodyFile("getFirstTenUsersResponse.json")
+                )
+        );
+
+
+        // make request and parse the json response
+        JsonNode userResponse = parseJson(
+            given()
+                .port(4000)
+                .get("/api/manager/secure/user?page=1")
+            .then()
+                .extract()
+                .response()
+                .asString()
+        );
+
+        // make sure the response matches the saved snapshot
+        assertThat(userResponse, matchesSnapshot());
+    }
+
+    /**
+     * Make sure a user can be created
+     */
+    @Test
+    public void canCreateUser() throws IOException {
+        // create wiremock stub for create users endpoint
+        stubFor(
+            post(urlPathEqualTo("/api/v2/users"))
+                .withRequestBody(matchingJsonPath("$.email", equalTo("test-new-user@test.com")))
+                .willReturn(
+                    aResponse()
+                        .withBodyFile("createNewUserResponse.json")
+                )
+        );
+
+
+        // make request and parse the json response
+        JsonNode createUserResponse = parseJson(
+            given()
+                .port(4000)
+                .body("{\n" +
+                    "  \"email\" : \"test-new-user@test.com\",\n" +
+                    "  \"password\" : \"password\",\n" +
+                    "  \"permissions\" : {}\n" +
+                    "}")
```

Comment:
I'd really like to see these JSON objects created with Jackson. I think I provided an example in another recent PR.